routes/tts_routes.py

```
@@ -0,0 +1,305 @@
+
+        audio_id = fm.pop(FM_AUDIO_ID, "")
+        for k in (FM_AUDIO, FM_AUDIO_DURATION, FM_AUDIO_FORMAT):
+            fm.pop(k, None)
+
+        save_ok, save_msg, _new_mtime = registry.post_service.save_file(
+            article_path, body, frontmatter_data=fm
+        )
+        if not save_ok:
+            return jsonify({"success": False, "message": f"清除 audio 字段失败: {save_msg}"}), 500
+
+        # 通知插件删除托管音频（若有凭证）
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)